src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/DraftAdjudicationDto.kt

```diff
@@ -20,6 +20,8 @@ data class DraftAdjudicationDto(
   val createdByUserId: String? = null,
```

Comment:
I am going to remove these in another PR, once I have updated the UI, as they just cause confusion.

Reply:
Sorry - is createdBy going to removed in preference to startedBy?

Reply:
Yep - have a variable with the name `createdByUserId` would be very confusing, as that is already used in the DB audit tables - and that is not what we want here